addon/components/utils/utm-link-builder.hbs

```diff
@@ -10,34 +10,79 @@
-      <span class="font-color-gray-500 text-size-5">{{t "utms.fields.medium"}}</span>
-      <OSS::InputContainer @value={{this.utmMedium}}
-                           @placeholder={{t "utms.fields.medium_placeholder"}}
-                           {{on "keyup" (fn this.notifyChanges "utmMedium")}}
-                           data-control-name="utm_medium_input" />
-      <span class="font-color-gray-500 text-size-5">{{t "utms.fields.campaign"}}</span>
-      <OSS::InputContainer @value={{this.utmCampaign}}
-                           @placeholder={{t "utms.fields.campaign_placeholder"}}
-                           {{on "keyup" (fn this.notifyChanges "utmCampaign")}}
-                           data-control-name="utm_campaign_input" />
+      {{#if @variablesEnabled}}
+        <Utils::TemplatedInputGroup
+          @title={{t "utms.fields.source"}}
+          @variables={{@variables}}
+          @onChange={{this.onUtmSourceChange}}
+          @value={{this.utmSource}}
+          @placeholder={{t "utms.fields.source_placeholder"}}
+          @required={{true}}
+          data-control-name="utm_source_input"
+          @id="utm_source_input"
+        />
+        <Utils::TemplatedInputGroup
+          @title={{t "utms.fields.medium"}}
+          @variables={{@variables}}
+          @onChange={{this.onUtmMediumChange}}
+          @value={{this.utmMedium}}
+          @placeholder={{t "utms.fields.medium_placeholder"}}
+          @required={{true}}
+          data-control-name="utm_medium_input"
+          @id="utm_medium_input"
+        />
+        <Utils::TemplatedInputGroup
+          @title={{t "utms.fields.campaign"}}
+          @variables={{@variables}}
+          @onChange={{this.onUtmCampaignChange}}
+          @value={{this.utmCampaign}}
+          @placeholder={{t "utms.fields.campaign_placeholder"}}
+          @required={{true}}
+          data-control-name="utm_campaign_input"
+          @id="utm_campaign_input"
+        />
+      {{else}}
+        <span class="font-color-gray-500 text-size-5">{{t "utms.fields.source"}}</span>
+        <OSS::InputContainer
+          @value={{this.utmSource}}
+          @placeholder={{t "utms.fields.source_placeholder"}}
+          {{on "keyup" (fn this.notifyChanges "utmSource")}}
+          data-control-name="utm_source_input"
+        />
+        <span class="font-color-gray-500 text-size-5">{{t "utms.fields.medium"}}</span>
+        <OSS::InputContainer
+          @value={{this.utmMedium}}
+          @placeholder={{t "utms.fields.medium_placeholder"}}
+          {{on "keyup" (fn this.notifyChanges "utmMedium")}}
+          data-control-name="utm_medium_input"
+        />
+        <span class="font-color-gray-500 text-size-5">{{t "utms.fields.campaign"}}</span>
+        <OSS::InputContainer
+          @value={{this.utmCampaign}}
+          @placeholder={{t "utms.fields.campaign_placeholder"}}
+          {{on "keyup" (fn this.notifyChanges "utmCampaign")}}
+          data-control-name="utm_campaign_input"
```

Comment:
i know it's from the old code but can you replace the `text-size-5` w/ `font-size-md` as we are here ^^ ?